app/waf/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build with:
+#   docker build . -t f5devcentral/f5-wafaas-openstack:latest
+
+# upload image with (authorized user only)
+#   docker push f5devcentral/f5-wafaas-openstack:latest
+
+FROM node:11.6.0
+
+LABEL maintainer="F5 Networks Inc. Corp."
+EXPOSE 3000/tcp
+
+WORKDIR /root/app_dir
+ADD dist dist
```

Comment:
Do you assume that compiling work should be done before building Docker image? Will it be better, if we compile the code during building Docker image?

Reply:
I suppose travis does the 2-phase jobs: first `npm build`, second: `docker build`. 
benefit for this is `npm build` can run on the host (instead of container) before generating the image.